control-userlayer/src/main/java/org/oskari/control/userlayer/EditUserLayerHandler.java

```diff
@@ -0,0 +1,91 @@
+package fi.nls.oskari.control.data;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import fi.mml.map.mapwindow.util.OskariLayerWorker;
+import fi.nls.oskari.annotation.OskariActionRoute;
+import fi.nls.oskari.control.ActionDeniedException;
+import fi.nls.oskari.control.ActionException;
+import fi.nls.oskari.control.ActionHandler;
+import fi.nls.oskari.control.ActionParameters;
+import fi.nls.oskari.control.ActionParamsException;
+import fi.nls.oskari.domain.map.userlayer.UserLayer;
+import fi.nls.oskari.domain.map.userlayer.UserLayerStyle;
+import fi.nls.oskari.service.OskariComponentManager;
+import fi.nls.oskari.util.ConversionHelper;
+import fi.nls.oskari.util.JSONHelper;
+import fi.nls.oskari.util.ResponseHelper;
+import org.oskari.map.userlayer.service.UserLayerDataService;
+import org.oskari.map.userlayer.service.UserLayerDbService;
+
+/**
+ * Expects to get layer id as http parameter "id".
+ */
+@OskariActionRoute("EditUserLayer")
+public class EditUserLayerHandler extends ActionHandler {
+    private final static String PARAM_ID = "id";
```

Comment:
This could be static final for consistency with the rest of them and the rest of the codebase